astroquery/nist/core.py

```diff
@@ -85,7 +85,8 @@ def _args_to_payload(self, *args, **kwargs):
 
         """
         request_payload = {}
-        request_payload["spectra"] = kwargs['linename']
+        linename = kwargs["linename"]
+        request_payload["spectra"] = linename if isinstance(linename, str) else " ".join(linename)
```

Comment:
@eerovaher 's changes has been applied. I can add an example to the docs if that what yall want, and changelog if needed